spring-cloud-data-admin/src/main/resources/META-INF/spring-cloud-data-admin/admin.properties

```diff
@@ -11,3 +11,18 @@ spring.yarn.client.files.0=file:${clouddata.yarn.app.appmaster.path:.}/spring-cl
 spring.yarn.client.files.1=file:${clouddata.yarn.app.container.path:.}/spring-cloud-data-yarn-container-@project.version@.jar
 spring.yarn.client.launchcontext.archiveFile=spring-cloud-data-yarn-appmaster-@project.version@.jar
 spring.yarn.client.resource.memory=1g
+
+# CF Deployer configuration
```

Comment:
All these properties need to have have `=` instead of `:` as the assignment operator. With the latest PR, this will change anyway.